draft-birkholz-rats-corim.md

```diff
@@ -178,6 +178,38 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
 
 Privacy Considerations
 
+# Concise SWID Data Extension
+
+# 2.7 The link-entry Map
+
+The link-entry map has been extended to define new relationship between a
+CoSWID and CoMID
+
+When defining CoSWID to CoMID link entry map the following optional elements
+should not be used:
+* artifact
+* ownership
+* use
+
```

Comment:
The ISO SWID TAGs spec also allows use of IANA link relations IANA (http:// www .iana .org/ assignments/ link -relations/ link -relations .xhtml) which we want to disallow for use with CoSWID to CoMID.

Reply:
Ok, I will add this specific clarification in the text!